src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -7,6 +7,7 @@ import { Switch } from 'antd';
 import 'react-table/react-table.css';
 import IconicLink from '../../components/buttons/IconicLink';
 import { GuppyConfigType, TableConfigType } from '../configTypeDef';
+import { dicomViewerPath } from '../../localconf';
```

Comment:
if this will only be used by data explorer, we can put this value into explorer config as well rather than importing from localconf to simplifies code

Reply:
oh ok, sure i can do that